sceptre/hooks/cmd.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+from sceptre.hooks import Hook
+from sceptre.exceptions import InvalidHookArgumentTypeError
+
+
+class Cmd(Hook):
+    """
+    Cmd implements a Sceptre hook which can run arbitrary commands.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(Cmd, self).__init__(*args, **kwargs)
+
+    def run(self):
+        """
+        Runs the argument string in a subprocess.
+
+        :raises: sceptre.exceptions.InvalidTaskArgumentTypeException
+        :raises: subprocess.CalledProcessError
+        """
+        try:
+            subprocess.check_call(self.argument, shell=True)
```

Comment:
Is that 'did we' or 'why did we'?

- did we: yes
- why did we: it's gives us access to constructs defined at the shell level, such as piping. It's considered a security risk if you pass unfiltered external input to `subprocess.check_call` but we decided it was alright here because it'd only run in an environment where the user already had access.